Packs/PrismaCloud/Integrations/RedLock/RedLock.yml

```diff
@@ -689,14 +979,188 @@ script:
+      name: filter_type
+      predefined:
+      - to_now
+      - absolute
+      - relative
+      required: false
+      secret: false
+    - default: false
+      defaultValue: '1'
+      description: Number of time units
+      isArray: false
+      name: filter_time_amount
+      required: false
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      defaultValue: day
+      description: Time unit
+      isArray: false
+      name: filter_time_unit
+      predefined:
+      - epoch
+      - login
+      - hour
+      - day
+      - week
+      - month
+      - year
+      required: false
+      secret: false
+    - default: false
+      description: 'Start time , for example: 11/01/2021 10:10:10'
+      isArray: false
+      name: filter_start_time
+      required: false
+      secret: false
+    - default: false
+      description: End time in Unix time (the number of seconds that have elapsed
+        since the Unix epoch) for the absolute time type
+      isArray: false
+      name: filter_end_time
+      required: false
+      secret: false
+    - default: false
+      description: Asset type to search with
+      isArray: false
+      name: filter_asset_type
+      required: false
+      secret: false
+    - default: false
+      description: Asset name to search with
+      isArray: false
+      name: filter_asset_name
+      required: false
+      secret: false
+    - default: false
+      description: 'User to filter with, example: ayman@example.domain'
+      isArray: false
+      name: filter_user
+      required: false
+      secret: false
+    - default: false
+      description: 'Status to filter with, example: passed'
+      isArray: false
+      name: filter_status
+      predefined:
+      - ''
+      required: false
+      secret: false
+    deprecated: false
+    description: List DevOps Scans
+    execution: false
+    name: redlock-list-scans
+    outputs:
+    - contextPath: Redlock.Scans.id
+      description: Scan ID
+      type: String
+    - contextPath: Redlock.Scans.attributes.user.[0]
+      description: Scan user
+      type: String
+    - contextPath: Redlock.Scans.attributes.status
+      description: Scan status
+      type: String
+  - arguments:
+    - default: false
+      description: The scan ID
+      isArray: false
+      name: scan_id
+      required: true
+      secret: false
+    deprecated: false
+    description: Get scan status
+    execution: false
+    name: redlock-get-scan-status
+    outputs:
+    - contextPath: Redlock.Scans.id
+      description: Scan ID
+      type: String
+    - contextPath: Redlock.Scans.attributes.status
+      description: Scan status
+      type: String
+  - arguments:
+    - default: false
+      description: The scan ID
+      isArray: false
+      name: scan_id
+      required: true
+      secret: false
+    deprecated: false
+    description: Get scan results
+    execution: false
+    name: redlock-get-scan-results
+    outputs:
```

Comment:
Some of the output's contextPaths (in the 3 new commands) seems not accurate.  
In order to regenerate the outputs contextPaths please run - demisto-sdk generate-outputs.
See:  [generate-outputs](https://github.com/demisto/demisto-sdk/tree/master/demisto_sdk/commands/generate_outputs#generate-outputs)

Reply:
I have fixed that